api/src/pcapi/core/educational/adage_backends/logger.py

```diff
@@ -101,22 +101,22 @@ def get_cultural_partner(self, siret: str) -> venues_serialize.AdageCulturalPart
         logger.info("Adage has been called at %s", f"{self.base_url}/v1/etablissement-culturel/{siret}")
         if siret == "12345678200010":
             return venues_serialize.AdageCulturalPartner(
-                id="128028",
+                id="128028",  # type: ignore [arg-type]
                 venueId=None,
                 siret=siret,
                 regionId=None,
                 academieId=None,
                 statutId=3,
                 labelId=None,
-                typeId="8",
+                typeId="8",  # type: ignore [arg-type]
                 communeId="26324",
                 libelle="Fête du livre jeunesse de St Paul les trois Châteaux",
                 adresse="Place Charles Chausy",
                 siteWeb="http://www.fetedulivrejeunesse.fr/",
-                latitude="44.350457",
-                longitude="4.765918",
-                actif="1",
-                dateModification="2021-09-01 00:00:00",
+                latitude="44.350457",  # type: ignore [arg-type]
+                longitude="4.765918",  # type: ignore [arg-type]
+                actif="1",  # type: ignore [arg-type]
+                dateModification="2021-09-01 00:00:00",  # type: ignore [arg-type]
```

Comment:
datetime : 

```suggestion
                dateModification=datetime.datetime.(2021, 9, 1),
```

Mêmes modifications dans `src/pcapi/core/educational/adage_backends/testing.py`.